exist-core/src/main/java/org/exist/xupdate/Modification.java

```diff
@@ -206,12 +205,12 @@ protected void declareVariables(XQueryContext context) throws XPathException {
 	 */
 	protected void declareNamespaces(XQueryContext context) throws XPathException {
 		Map.Entry<String, String> entry;
-		for (final Iterator<Map.Entry<String, String>> i = namespaces.entrySet().iterator(); i.hasNext();) {
-			entry = (Map.Entry<String, String>) i.next();
-			context.declareNamespace(
-				entry.getKey(),
-				entry.getValue());
-		}
+        for (Map.Entry<String, String> stringStringEntry : namespaces.entrySet()) {
+            entry = stringStringEntry;
+            context.declareNamespace(
+                    entry.getKey(),
+                    entry.getValue());
+        }
```

Comment:
I think this needs tidying up. `entry` is redundant, and then `stringStringEntry` should be renamed as `entry`.

Reply:
ah... good catch; i will fix